src/main/java/com/tejaaa/RevisionSchedulerService/service/UserLearningItemServiceImpl.java

```diff
@@ -38,8 +38,11 @@ public class UserLearningItemServiceImpl implements UserLearningItemService{
 
     @Override
     public List<UserLearningItem> getAllUserLearningItems() {
-        log.info("fetching all UserLearningItem's");
-        List<UserLearningItem> response =  userLearningRepo.findAll();
+        AppUser user = appUserRepo.findByUsername(SecurityContextHolder.getContext().getAuthentication().getName());
```

Comment:
move user object to class level as this will be required at multiple places in future